src/extractPdfText.ts

```diff
@@ -1,51 +1,59 @@
 import { getPdfPagesCount } from "./getPdfPagesCount";
 import { capitalizeFirstLetter } from "./helperFunctions";
-const pdfLib = require('pdf-to-text');
-// figure out how to deal with this import vs require thing
-// tried modifying tsconfig.json target and module to es2022 and moduleResolution uncomment
-// ^ but didn't work, kept saying module couldn't be found, etc.
+import { join } from "path";
+//@ts-expect-error no type declarations for this module
+import pdfLib from "pdf-to-text";
+import { PdfLib } from "./types";
 
 // Get the user's desired pages range to translate, with a few validity checks
-const getPagesRange = async(pageFrom?: number, pageTo?: number) => {
-    const pagesCount = await getPdfPagesCount();
+const getPagesRange = async (pageFrom = 1, pageTo?: number) => {
```

Comment:
`pageFrom = 1` will automatically default to 1 is it is omitted when passed as an argument. TS will automatically infer `?: number`